pkgs/applications/misc/gitmux/default.nix

```diff
@@ -0,0 +1,38 @@
+{ fetchFromGitHub, buildGoModule, lib, testVersion, gitmux }:
+
+buildGoModule rec {
+  pname = "gitmux";
+  version = "0.7.5";
+
+  src = fetchFromGitHub {
+    owner = "arl";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0v9w5l1r5myi26mci2gn9pfii7q11qdd8bir6biy205ls966czqy";
+  };
+
+  vendorSha256 = "sha256-cgQg/ZyKlEQfayoETeKWTEidhvt8dsxrHfPYLgdv8gY=";
+
+  # GitHub source does contain a regression test for the module
+  # but it requires networking as it git clones a repo from github
+  doCheck = false;
+
+  prePatch = ''
+    substituteInPlace gitmux.go --replace '<<development version>>' "${version}"
```

Comment:
This `prePatch` should be replaced with a `ldflags = [ "-X main.version=${version}" ];` line.

Reply:
Thanks for review, change implemented.